src/webapp/helpers/time.js

```diff
@@ -1,8 +1,15 @@
-import { differenceInDays, fromUnixTime } from "date-fns";
+import { differenceInDays, parseISO } from "date-fns";
 
-const daysSinceSlackMessage = (slackTs) => {
-  const datePosted = fromUnixTime(Number(slackTs));
+/**
+ * @param {string} isoTimestamp - ISO timestamp, eg "2020-10-09T18:38:28.000Z"
+ * @returns {number} - difference in days since provided ISO timestamp
+ */
+const getDaysSinceIsoTimestamp = (isoTimestamp) => {
+  if (!isoTimestamp) {
+    return -1;
+  }
```

Comment:
I'm unable to view the prod Airtable, but based on how many rows were missing the `Date Changed to Delivery Needed` field in Staging, I thought this would be a prudent safety check 🤷‍♂️